plugins/periskop-backend/src/types.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface AggregatedError {
```

Comment:
just mentioning that it's a pattern for some to make a `-common` isomorphic package with shared types and stuff (i see this appearing in the other package too). No rush, some duplication can be perfectly fine and less stuff to manage, but thought it was worth mentioning

Reply:
Yes, thanks - I will keep this for a follow-up, trying to keep the changeset as small as possible here as we replace our internal implementation